spectrocloud/cluster_common_virtual_machine.go

```diff
@@ -196,11 +195,11 @@ func prepareVolume(d *schema.ResourceData) []*models.V1VMVolume {
 		}
 		return vmVolumes
 	} else {
-		return prepareDefaultVolume(d)
+		return prepareDefaultVolumeSpec(d)
 	}
 }
 
-func prepareDefaultNetwork() []*models.V1VMNetwork {
+func prepareDefaultNetworkSpec() []*models.V1VMNetwork {
```

Comment:
please move everything related to network spec into separate file.

Reply:
done